server.js

```diff
@@ -277,7 +283,7 @@ Server.prototype.onWebSocketConnection = function (socket) {
   socket.on('close', self._onWebSocketClose.bind(self, socket))
 }
 
-Server.prototype._onWebSocketRequest = function (socket, params) {
+Server.prototype._onWebSocketRequest = function (socket, params, onWebSocketRequestCallback) {
```

Comment:
This parameter will always be `undefined`. Where is a function ever passed in? What's the point of this?